framework-cloud/cloud-deployment-plugin/pom.xml

```diff
@@ -38,5 +38,10 @@
       <artifactId>maven-plugin-annotations</artifactId>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
```

Comment:
Should we directly add OpenShift dependency to the plugin?
Or we should rather leave this to the plugin users? So every user will add cloud dependency to the plugin based on his choice?